compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -536,7 +536,8 @@ private void addRequiredParamsToGeneratedInitFunction(BLangFunction initFunction
                     ASTBuilderUtil.createVariable(initFunction.pos,
                                                   requiredParameter.name.getValue(), requiredParameter.type,
                                                   createRequiredParamExpr(requiredParameter.expr),
-                                                  new BVarSymbol(0, names.fromString(requiredParameter.name.getValue()),
+                                                  new BVarSymbol(Flags.asMask(requiredParameter.flagSet),
+                                                  names.fromString(requiredParameter.name.getValue()),
```

Comment:
Let's fix the alignment here - align this with the start of `Flags.asMask(requiredParameter.flagSet),` similar to the rest of the args below.

Reply:
Fixed.